### PR TITLE
[NO-ISSUE] Rearrange tests to be more like source structure.

### PR DESCRIPTION
**Pull request by austinkelleher**
The deletions were a few accidental duplicate tests.

### CODE REVIEW

File: test/unit/dao/Entity-test.js [hunk 1]
```diff
@@ -15,15 +15,3 @@ test('should allow passing id', (t) => {
   const entity = new Entity({ id: id })
   t.is(entity.getId(), id)
 })
-
-test('should allow fetching id using getId', (t) => {
```
Comment (austinkelleher) [inline test/unit/dao/Entity-test.js]: This is old code and should be removed.

File: test/unit/getTableName-test.js [hunk 2]
```diff
@@ -1,19 +0,0 @@
-require('require-self-ref')
-
```
Comment (Druotic) [inline test/unit/getTableName-test.js]: Huh, were there two of these files or is github doing weird things? Looks like one was deleted, one was renamed?

Comment (austinkelleher) [inline test/unit/getTableName-test.js]: I put in the PR description: "The deletions were a few accidental duplicate tests."

Comment (Druotic) [inline test/unit/getTableName-test.js]: Oh, oops. Descriptions? What are those?

**Pull request merged**